packages/docusaurus-utils/src/globUtils.ts

```diff
@@ -68,7 +69,7 @@ export function createAbsoluteFilePathMatcher(
 
   function getRelativeFilePath(absoluteFilePath: string) {
     const rootFolder = rootFolders.find((folderPath) =>
-      absoluteFilePath.startsWith(folderPath),
+      absoluteFilePath.startsWith(addTrailingSlash(folderPath)),
```

Comment:
I'm unsure between `addTrailingSlash` and `addTrailingPathSeparator`. Tentatively went with `addTrailingSlash`; no one reported the bug before anyway so it won't be a regression

Reply:
OK, never mind: Windows tests tell me that `addTrailingPathSeparator` is the right one...